.github/dependabot.yaml

```diff
@@ -1,7 +1,30 @@
 version: 2
 enable-beta-ecosystems: true
 updates:
-  - package-ecosystem: "pub"
+  - package-ecosystem: "github-actions"
     directory: "/"
     schedule:
-      interval: "weekly"
\ No newline at end of file
+      interval: "weekly"
```

Comment:
What does this do?

Reply:
checks outdated `runs-on` machine.

checks steps that `uses` , for example
- uses: actions/checkout@v3
- uses: subosito/flutter-action@v2
when ever there are issues or outdated ones.